### ISSUE TITLE
Add buttons to GUI to open relevant sections of the System Settings preferences panel [ENHANCEMENT]

### ISSUE THREAD

**Issue by KieranConlon** [intitni/CopilotForXcode#53]
## Add button to open System Settings - Accessibility API
The installation intructions for granting permissions requires that the app needs Accessibility permissions to work properly.
Suggestion to simplify access to the correct setting in System Settings by adding a button/control on the GUI to run a `bash` command to open System Settings on the correct panel.

On macOS Ventura (and I think Monterrey too) the command is:
``` bash
open "x-apple.systempreferences:com.apple.preference.security?Privacy_Accessibility"
```

## Add button to open System Settings - Xcode Source Editor Extension
The installation intructions for enabling the extension requires that the Copilot is added as an Xcode extension.
Suggestion to simplify access to the correct setting in System Settings by adding a button/control on the GUI to run a `bash` command to open System Settings on the correct panel.

On macOS Ventura the command is:
``` bash
open "x-apple.systempreferences:com.apple.ExtensionsPreferences"  
```

NOTE:  I don't have a macOS Monterrey install, these steps should be checked & verified.
On macOS Monterrey, access to the Extensions preference panel is not supported by URLScheme, may need to use AppleScript instead:
``` AppleScript
tell application "System Preferences"
	activate
	set the current pane to pane id "com.apple.preferences.extensions"
end tell
```

or using `bash`
``` bash
osascript -e 'tell Application "System Preferences"' -e 'activate' -e 'set the current pane to pane id "com.apple.preferences.extensions"' -e 'end tell'
```

---
List of methods for opening System Settings obtained from [bvanpeski - SystemPreferences](https://github.com/bvanpeski/SystemPreferences)

**Comment by dustturtle** [comment]
why I can not see copilot for xcode in extensions? mac os 12.6

**Comment by dustturtle** [comment]
@intitni

**Comment by intitni** [comment]
@dustturtle Please do not reply to a random issue with your own question. Please check #42 to see if it can solve your problem.

**Comment by intitni** [comment]
Released in 0.16.0
